config/config.go

```diff
@@ -24,6 +25,13 @@ type HAproxyConfig struct {
 	UseHostnames bool   `envconfig:"USE_HOSTNAMES"`
 }
 
+type EnvoyConfig struct {
+	UseGRPCAPI   bool   `envconfig:"USE_GRPC_API" default:"false"`
```

Comment:
You'll have to set `ENVOY_USE_GRPC_API=true` for this functionality to be enabled

Reply:
Can't we just run both all the time?

Reply:
I was thinking that people who want to use Sidecar with HAProxy or prefer sticking with the Envoy REST API shouldn't take the performance hit of composing the Envoy updates for the gRPC API on each state change. I can make this setting opt out rather than opt in if you'd prefer that.

Reply:
I think that's better